relay-general/src/pii/processor.rs

```diff
@@ -426,6 +428,36 @@ mod tests {
         assert_annotated_snapshot!(event);
     }
 
+    #[test]
+    fn test_span_default_rules() {
+        let ds_config = DataScrubbingConfig {
+            scrub_data: true,
+            scrub_defaults: true,
+            ..Default::default()
+        };
+        let config = ds_config.pii_config().unwrap().as_ref().unwrap();
+
+        let span = Annotated::new(Span {
+            timestamp: Annotated::new(Utc.ymd(1970, 1, 1).and_hms_nano(0, 0, 0, 0).into()),
+            start_timestamp: Annotated::new(Utc.ymd(1968, 1, 1).and_hms_nano(0, 0, 0, 0).into()),
+            description: Annotated::new(
+                "desc with http://example.com/userid=1&token=1234 the end".to_owned(),
+            ),
+            trace_id: Annotated::new(TraceId("4c79f60c11214eb38604f4ae0781bfb2".into())),
+            span_id: Annotated::new(SpanId("fa90fdead5f74052".into())),
+            status: Annotated::new(SpanStatus::Ok),
+            ..Default::default()
+        });
+        let mut event = Annotated::new(Event {
+            spans: Annotated::new(vec![span]),
+            ..Default::default()
+        });
+
+        let mut processor = PiiProcessor::new(config.compiled());
+        process_value(&mut event, &mut processor, ProcessingState::root()).unwrap();
+        assert_annotated_snapshot!(event);
```

Comment:
nit: we're only interested in ensuring the description is `[Filtered]`, so it may be easier to read that assertion over a snapshot?

Reply:
The test is to ensure that the things are working as expected now, and it will show the problem if we have the regression. But I do not really care who this implemented.